pkg/reconciler/common/testdata/kodata/tekton-addon/0.0.2/01-clustertriggerbindings/bitbucket-1.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: ClusterTriggerBinding
+metadata:
+  name: bitbucket-pullreq-0.0.2-copy
```

Comment:
why duplicated file ?

Reply:
I was testing having multiple files in the directory